testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveStorageFormats.java

```diff
@@ -1156,4 +1266,20 @@ public String getReadType(HiveTimestampPrecision precision)
     {
         return io.trino.jdbc.Row.builder();
     }
+
+    private static class StorageFormatHiveCompatibilityColumnData
```

Comment:
You can shorten the class name, by removing `StorageFormat`. 
In fact, the class is agnostic to storage format, so the name doesn't need to include that.